package.json

```diff
@@ -70,6 +77,16 @@
 					"buf.work",
 					"buf.gen"
 				]
+			},
+			{
+				"id": "proto",
+				"extensions": [
+					".proto"
+				],
+				"aliases": [
+					"Protocol Buffers",
+					"Protobufs"
```

Comment:
If we think that the `aliases` are necessary, I'd probably shorten this to "Protobuf" - I always thought that "Protobufs" is a bit of a misnomer.

Reply:
I don't think they're necessary - IIUC this just improves discoverability (and isn't strictly related to this PR). Changed it to `Protobuf`, but lmk if you'd rather me remove it.

Reply:
Sweet, leaving them in seems fine - better discoverability is a good thing.